### PR TITLE
fix: improvements from docs review

### PR DESCRIPTION
**Pull request by vanbasten17**
_Remember to tag the PR with **WIP** tag if it's not ready to be merged_.  
[Reference](https://blog.codeminer42.com/on-writing-a-great-pull-request-37c60ce6f31d/)

## Description
* Improve `index.d.ts` with definitions.
* Pass missing `animationsEnabled` prop.

## Context
Added while doing documentation review in: https://github.com/hubtype/botonic/pull/822

### CODE REVIEW

File: packages/botonic-react/src/components/webchat-settings.jsx [hunk 1]
```diff
@@ -70,6 +75,10 @@ export const normalizeWebchatSettings = settings => {
     if (!theme.userInput.enable) theme.userInput.enable = {}
     theme.userInput.enable = enableUserInput
   }
+  if (enableAnimations !== undefined) {
+    if (!theme.animations.enable) theme.animations.enable = {}
```
Comment (dpinol) [inline packages/botonic-react/src/components/webchat-settings.jsx:79]: line is unnecessary

**Review by dpinol — APPROVED** [review]
amazing typing! :-)

File: packages/botonic-react/src/components/webchat-settings.jsx (hunk 1 above)
Comment (vanbasten17) [reply packages/botonic-react/src/components/webchat-settings.jsx:79]: why?

**Review by ericmarcos — APPROVED** [review]

**Review by dpinol — COMMENTED** [review]
I forgot to submit my review :-)
can you remove the line in the next botonic PR you work on?

thanks

File: packages/botonic-react/src/components/index.d.ts [hunk 2]
```diff
@@ -74,4 +74,104 @@ export const Subtitle: React.FunctionComponent<SubtitleProps>
...
+    persistentMenu?: PersistentMenuProps
+    blockInputs?: BlockInputOption[]
+    menu?: { darkBackground?: boolean } & CustomProp
+    menuButton?: CustomProp
+  } & EnableProp &
+    StyleProp
+}
+
+export interface WebchatSettingsProps {
+  theme?: ThemeProps
```
Comment (AinaVendrell) [inline packages/botonic-react/src/components/index.d.ts:166]: Should `coverComponent` be included here?

Comment (vanbasten17) [reply packages/botonic-react/src/components/index.d.ts:166]: You are right Aina, I think the PR regarding CoverComponent wasn't merged by the time I typed these properties. Feel free to create a new PR and adding it :) Reach me if you have any doubt

**Pull request merged**